File: java/factorial.java
class factorial
{
  void main()
  {
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();
    int f=1;
    for (int i=2;i<=n;i++)
    { 
        f=f*i;
    }
    System.out.println(f);
  }
}
    

File: Kotlin/Factorial.kt
fun main(args: Array<String>) {

    println("Enter the value you want to find factorial of: ")
    val factorialOf: String = readLine().orEmpty()
    var num = factorialOf.toInt()
    var result = 1

    while (num > 0) {
        result *= num
        num--
    }

    println("The factorial of $factorialOf is: $result")
}